src/main/java/com/exadel/discountwebapp/discount/service/DiscountService.java

```diff
@@ -43,9 +45,17 @@
     public DiscountResponseVO findById(Long id) {
```

Comment:
let's rename this method to reflect the update of the Discount -> findByIdAndUpdateStatistics

Reply:
fixed